src/components/CcyInfoModal/CcyInfoModal.js

```diff
@@ -13,13 +13,33 @@ const CcyInfoModal = ({
   fetchCcyArticle,
   article,
 }) => {
+  const { body, title } = article
+
+  const containerRef = useRef()
   useEffect(() => {
     if (isModalVisible && _isEmpty(article)) {
       fetchCcyArticle()
     }
   }, [isModalVisible])
 
-  const { body, title } = article
+  useEffect(() => {
+    if (!body || !containerRef.current) {
+      return
+    }
+    const links = [...containerRef.current.querySelectorAll('a')]
```

Comment:
do we really need spread here? as far as I know, browsers should support `forEach` iterations on `NodeList` out of the box